packages/react-components/src/components/knowledge-graph/KnowledgeGraphPanel.tsx

```diff
@@ -208,31 +213,32 @@ export const KnowledgeGraphContainer: React.FC<KnowledgeGraphInterface> = ({
   return (
     <Container header={<Header variant='h3'>Knowledge Graph</Header>}>
       <div style={{ width: '100%', height: '100%' }}>
-        <SpaceBetween direction='vertical' size='s'>
-          <SpaceBetween direction='horizontal' size='s'>
-            <Input
-              type='text'
-              value={searchTerm}
-              onChange={(e) => {
-                setSearchTerm(e.detail.value);
-              }}
-            ></Input>
-            <Button onClick={onSearchClicked} data-testid='search-button'>
-              <FormattedMessage
-                id='KnowledgeGraphPanel.button.search'
-                defaultMessage='Search'
-                description='Search button text'
-              />
-            </Button>
-          </SpaceBetween>
-          {/* inline styling here for testing only this will be fixed in the next PR */}
+        <SpaceBetween direction='vertical' size='xl'>
+          <div style={{ minWidth: '500px' }}>
+            <Grid gridDefinition={[{ colspan: 9 }, { colspan: 3 }]}>
+              <Input
+                type='search'
+                value={searchTerm}
+                onChange={(e) => {
+                  setSearchTerm(e.detail.value);
+                }}
+              ></Input>
+              <Button onClick={onSearchClicked} data-testid='search-button'>
+                <FormattedMessage
+                  id='KnowledgeGraphPanel.button.search'
+                  defaultMessage='Search'
+                  description='Search button text'
+                />
+              </Button>
+            </Grid>
+          </div>
           <div ref={containerRef} className={`${STYLE_PREFIX} ${className || ''}`.trim()} {...props}>
             <GraphView
               className={`${STYLE_PREFIX}-canvas`}
               ref={cy}
               stylesheet={stylesheet}
               elements={getElementsDefinition([...nodeData.values()], [...edgeData.values()])}
-              layout={edgeData.size > 0 ? breadthFirstlayout : coselayout}
+              layout={edgeData.size > 0 ? breadthFirstlayout : nodeData.size > 1 ? gridlayout : presetlayout}
```

Comment:
Same question here about `>` vs `>==`.

Reply:
yup we want to address one corner case which is when the number of nodes =1